supabase/migrations/20260816150923_split_mfa_session_and_email_otp_checks.sql

```
@@ -0,0 +1,221 @@
+  expires_at timestamp with time zone NOT NULL,
+  created_at timestamp with time zone DEFAULT now() NOT NULL
+);
+
+COMMENT ON TABLE public.platform_impersonation_sessions IS
+'Short-lived sessions minted by platform-admin /private/log_as. '
+'Allows verify_mfa during support spoof of MFA-enforced users.';
+
+ALTER TABLE public.platform_impersonation_sessions OWNER TO postgres;
+
+CREATE INDEX IF NOT EXISTS platform_impersonation_sessions_expires_at_idx
+  ON public.platform_impersonation_sessions (expires_at);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)